src/app/components/BookList/BookList.jsx

```diff
@@ -3,35 +3,34 @@ import PropTypes from 'prop-types';
 
 import About from '../About/About';
 import Book from '../Book/Book';
+import ListTitle from '../ListTitle/ListTitle';
 
 const BookList = ({
   isJsEnabled,
   picks,
   type,
   displayInfo,
   picksCount,
+  idPrefix,
 }) => {
   const renderBookItems = currentBooks => (
     currentBooks.length ?
       currentBooks.map(book => <Book key={book.slug} pick={book} isJsEnabled={isJsEnabled} />)
       : null
   );
 
-  const { displayDate = {}, displayAge } = displayInfo;
-  const booksfound = `${picksCount} Book${picksCount === 1 ? '' : 's'} Found`;
-
   return (
     <div className="booklist-section nypl-column-three-quarters">
-      <h2 id="list-title" tabIndex="0">
-        {displayDate.month} {displayDate.year} Picks for {displayAge}
-        <span>{booksfound}</span>
-      </h2>
+      <div className="list-title-container">
+        <ListTitle displayInfo={displayInfo} picksCount={picksCount} idPrefix={idPrefix} />
```

Comment:
This is not what I meant. Now you added `idPrefix` as a prop dependency for `<BookList>`. You already had the default prop value of `idPrefix` in  `<ListTitle>` as "". So you don't need `idPrefix=""` or `idPrefix={idPrefix}` initialized here at all.

Reply:
Yup just realized that and updated.